testdata/baselines/reference/submodule/compiler/nodeNextPackageSelfNameWithOutDir.errors.txt.diff

```diff
@@ -5,21 +5,19 @@
 -
 -
 -!!! error TS2209: The project root is ambiguous, but is required to resolve export map entry '.' in file 'package.json'. Supply the `rootDir` compiler option to disambiguate.
```

Comment:
This is missing because we currently aren't collecting resolution diagnostics (nor any program diagnostic).